prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -196,7 +196,22 @@ fi_ibv_rdm_tagged_zero_request(struct fi_ibv_rdm_tagged_request *request)
 void fi_ibv_rdm_tagged_print_request(char *buf,
 				     struct fi_ibv_rdm_tagged_request *request);
 
+typedef short fi_ibv_rdm_buf_status_t;
+#define BUF_STATUS_FREE		((fi_ibv_rdm_buf_status_t) 0)
+#define BUF_STATUS_BUSY		((fi_ibv_rdm_buf_status_t) 1)
+#define BUF_STATUS_RECVED	((fi_ibv_rdm_buf_status_t) 2)
```

Comment:
typedefs should be avoided unless there's an architectural reason for them.  Weren't these enums in earlier patches?